tests/bibo/test_internals.py

```diff
@@ -74,3 +74,43 @@ def test_complete_path(tmpdir):
     assert len(ms_subdir) == 1
     # Check for completion help string as basename
     assert ms_subdir[0] == (subp.strpath, subp.basename)
+
+
+def test_match_case():
+    assert internals.match_case("hello", "Hello World") == "Hello"
+    assert internals.match_case("WORLD", "Hello World") == "World"
+    with pytest.raises(ValueError):
+        internals.match_case("bibo", "Hello World")
+
+
+def test_highlight_text():
+    s1 = "hello world"
+    s2 = "hello {}w{}orld".format(internals._ansi_bold, internals._ansi_unbold)
```

Comment:
Maybe this should just be a utility, especially given the private-ish internal `internals._ansi_bold`. Like, `format_bold`

Reply:
I don't think I understand you. Do you mean changing `internals.highlight_text` to `internals._highlight_text`? Yeah, I'm happy with that. In general we are far from having any reliable API and can make most of `internals` private.

Reply:
I mean, moving this common line which wraps a string with `internals._ansi_bold` into a dedicated utility, if its going to be used all over the place like this.

Reply:
Done!